### PR TITLE
Loki: Update content for log context ui

### PR DESCRIPTION
**Pull request by ivanahuckova**
This PR updates the content for Loki log context UI.

image
image

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx [hunk 1]
```diff
@@ -186,7 +186,7 @@ export function LokiContextUi(props: LokiContextUiProps) {
               )}
               className={styles.rawQuery}
             />
-            <Tooltip content="Initially executed log context query is created from all labels defining the stream for the selected log line. Use editor bellow to customize log context query.">
+            <Tooltip content="Initially executed log context query is created from all labels defining the stream for the selected log line. Use editor below to customize log context query.">
```
Comment (JStickler) [inline public/app/plugins/datasource/loki/components/LokiContextUi.tsx:189]:
```suggestion
            <Tooltip content="The initial log context query is created from all labels defining the stream for the selected log line. Use the editor below to customize the log context query.">
```

**Review by JStickler — COMMENTED** [review]
[docs team] Some suggestions for rewording.

**Review by svennergr — APPROVED** [review]
LGTM, just merge as soon as you think it's ready 🚀

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Pull request merged**